src/renderer/components/inputs/previews/ImagePreview.tsx

```diff
@@ -42,6 +42,11 @@ const ImagePreview = memo(({ path, schemaId, id }: ImagePreviewProps) => {
     const [isCpu] = useIsCpu;
     const [isFp16] = useIsFp16;
 
+    const setWidth =
+        img && img.height > img.width ? `${200 * (img.width / img.height)}px` : '200px';
+    const setHeight =
+        img && img.width > img.height ? `${200 * (img.height / img.width)}px` : '200px';
```

Comment:
Why? You just implemented the `maxW` & `maxH` the `Image` element previously used.

Reply:
True. I wrote that before I realized I needed to wrap the element, guess it's not doing anything different.